core/config/project_settings.cpp

```diff
@@ -1114,7 +1114,7 @@ ProjectSettings::ProjectSettings() {
 
 	_add_builtin_input_map();
 
-	custom_prop_info["display/window/handheld/orientation"] = PropertyInfo(Variant::STRING, "display/window/handheld/orientation", PROPERTY_HINT_ENUM, "landscape,portrait,reverse_landscape,reverse_portrait,sensor_landscape,sensor_portrait,sensor");
+	custom_prop_info["display/window/handheld/orientation"] = PropertyInfo(Variant::INT, "display/window/handheld/orientation", PROPERTY_HINT_ENUM, "Landscape (Left Side Up),Portrait (Upright),Reverse Landscape (Right Side Up),Reverse Portrait (Upside Down),Sensor Landscape (Left or Right Side Up),Sensor Portrait (Upright or Upside Down),Sensor (Any Orientation)");
```

Comment:
I would remove the hint (in parentheses) as on Android the orientation is dependent on the [device's physical dimensions](https://developer.android.com/reference/androidx/browser/trusted/ScreenOrientation) (although it could also be overridden at the manufacturer's discretion.